feat: Add security/{threat, disclosure, verifying} (#119)
kubewarden/docs: docs/security/verifying-kubewarden.md

- x509 certificate extension for GHA, "github_workflow_repository": `kubewarden/*`

## Helm charts

You can find our Helm charts in our `https://` traditional Helm repository under
https://charts.kubewarden.io.

The same Helm charts are signed via Sigstore's keyless signing, and pushed to an
OCI repository that can hold both the Helm chart and its signature as OCI
artifacts.

Since Helm 3.8.0, Helm has support for OCI registries, but because of
constraints in them, they can't be searched via `helm`. You can find the
[list of charts in GitHub Container Registry](https://github.com/orgs/kubewarden/packages?tab=packages&q=charts).

To verify a Helm chart, you need `cosign` installed. Then execute the following
command, for example:

```
COSIGN_EXPERIMENTAL=1 cosign verify ghrc.io/kubewarden/charts/kubewarden-controller:0.4.6 | jq

Verification for ghcr.io/kubewarden/charts/kubewarden-controller:0.4.6 --
The following checks were performed on each of these signatures:
  - The cosign claims were validated
  - Existence of the claims in the transparency log was verified offline
  - Any certificates were verified against the Fulcio roots.

  <snipped json>
```

> You can then verify either manually or with Sigstore tools that the cert in the

Reviewer: ```suggestion
You may either verify manually or with Sigstore tools that the cert in the
```

returned json contains the correct issuer, subject, and
`github_workflow_repository` extensions.

## Container images

> Both our production images used in our Helm charts and our development images

Reviewer: ```suggestion
Both the production images used in our Helm charts and our development images
```
